### PR TITLE
feat: docs of gitlab pipeline action

### PR DESCRIPTION
**Pull request by matarpeles**
# Description

Docs for GitLab pipeline action 

## Changes description:

Reorganized the Port execution agent hierarchy:
-  The Port Agent is now a section under "Setup Backend and not under Webhook actions.
-  Added a new section "GitLab Pipelines Actions" - guides how to setup an action for GitLab pipelines in Port
- Added an example of how to setup the port agent for GitLab triggering in '/create-self-service-experiences/setup-backend/port-execution-agent/quickstart'

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+| `omitPayload`    | `boolean` | Flag to control whether to send [`port_payload`](#action-message-structure) JSON string as an additional parameter to the GitLab pipeline (default: `false`).                                                                                                                                                                                               | `true` or `false`                    |
+| `omitUserInputs` | `boolean` | Flag to control whether to send the user inputs of the Port action as [GitLab CI/CD variables](https://docs.gitlab.com/ee/ci/variables/) to the GitLab pipeline. <br></br> By default, the user inputs are passed as variables (default: `false`). <br></br> When disabled, you can still get the user inputs from the `port_payload` (unless omitted too). | `true` or `false`                    |
+
+## Further steps
+
+- Contact us through Intercom to set up a Kafka topic for your organization runs.
```
Comment (MPTG94) [inline docs/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md:114]:
```suggestion
- Contact us through Intercom to set up a Kafka topic for your organization.
```

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+| `groupName`      | `string`  | Defines the GitLab group name                                                                                                                                                                                                                                                                                                                               | `port-labs`                          |
+| `defaultRef`     | `string`  | The default ref (branch / tag name) we want the action to use. <br></br> `defaultRef` can be overrided dynamically,<br></br> by adding `ref` as user input. <br></br> If not set, the agent triggers `main` branch                                                                                                                                          | `main`                               | `port-docs` |
```
Comment (MPTG94) [inline docs/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md:108]:
```suggestion
| `defaultRef`     | `string`  | The default ref (branch / tag name) we want the action to use. <br></br> `defaultRef` can be overriden dynamically,<br></br> by adding `ref` as a user input. <br></br> If not set, the agent triggers the `main` branch                                                                                                                                          | `main`                               | `port-docs` |
```

**Review by MPTG94 — APPROVED** [review]
Fix comments and merge

File: docs/create-self-service-experiences/setup-backend/port-execution-agent/Installation.md
[the remark this answers is not in the document]
Comment (matarpeles) [reply docs/create-self-service-experiences/setup-backend/port-execution-agent/Installation.md:1]: Installation, I forgot to change it in the title of the page, fixed

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+| `omitPayload`    | `boolean` | Flag to control whether to send [`port_payload`](#action-message-structure) JSON string as an additional parameter to the GitLab pipeline (default: `false`).                                                                                                                                                                                               | `true` or `false`                    |
```
[the remark this answers is not in the document]
Comment (matarpeles) [reply docs/create-self-service-experiences/setup-backend/gitlab-pipeline/gitlab-pipeline.md:109]: changed it , being accurate it's payload , not a variable or a parameter

**Pull request closed without merging**